### PR TITLE
FIX: help-numExecutors_zh_CN.html and translation of # of executors

### PR DESCRIPTION
**Pull request by anxk**
### Submitter checklist

- [x] [Know translation specification](https://github.com/jenkins-zh/translation-spec/blob/master/specification.md)

### Desired reviewers

@jenkinsci/chinese-localization-sig

### CODE REVIEW

**Comment by anxk** [comment]
1. help-numExecutors_zh_CN.html文件中的一些描述可能不正确，例如agent的执行器数目不能等于0，该提交按照原英文描述进行了翻译；
2. 对于原来在agent中将 # of executor翻译为并发构建数，master节点中翻译为执行者数量进行修改，统一翻译为执行器数量；

**Comment by anxk** [comment]
agent执行器不能为零参照hudson/model/Slave.java

**Comment by anxk** [comment]
检查没通过，跟本次提交好像没关系，log如下：
----

[2019-12-02T18:35:43.679Z] [INFO] --- spotbugs-maven-plugin:3.1.11:check (spotbugs) @ localization-zh-cn ---

[2019-12-02T18:35:43.679Z] [INFO] BugInstance size is 2

[2019-12-02T18:35:43.679Z] [INFO] Error size is 0

[2019-12-02T18:35:43.679Z] [INFO] Total bugs: 2

[2019-12-02T18:35:43.679Z] [ERROR] Exceptional return value of java.io.File.delete() ignored in io.jenkins.plugins.localization_zh_cn.UpdateCenterAction.doRemove(StaplerResponse) [io.jenkins.plugins.localization_zh_cn.UpdateCenterAction] At UpdateCenterAction.java:[line 47] RV_RETURN_VALUE_IGNORED_BAD_PRACTICE

[2019-12-02T18:35:43.679Z] [ERROR] Unread field: io.jenkins.plugins.localization_zh_cn.UpdateCenterAction.CRT; should this field be static? [io.jenkins.plugins.localization_zh_cn.UpdateCenterAction] At UpdateCenterAction.java:[line 16] SS_SHOULD_BE_STATIC

File: core/src/main/resources/hudson/model/Node/help-numExecutors_zh_CN.html [hunk 1]
```diff
@@ -1,11 +1,10 @@
 <div>
-  这个值控制着Jenkins并发构建的数量.
-  因此这个值会影响Jenkins系统的负载压力.
-  使用处理器个数作为其值会是比较好的选择.
-
+  Jenkins 可以在此节点上执行并发构建的最大数目。
   <p>
-  增大这个值会使每个构建的运行时间更长,但是这能够增大整体的构建数量,因为当一个项目在等待I/O时它允许CPU去构建另一个项目.
-
+  在开始的时候，使用节点的 CPU 个数作为该值是一个不错的选择。一个较大的值将会使每个构建花费更多的时间，
```
Comment (LinuxSuRen) [inline core/src/main/resources/hudson/model/Node/help-numExecutors_zh_CN.html]: CPU 通常会使用`核数`来描述。

**Pull request merged**